chromium_src/chrome/browser/ui/webui/help/version_updater_mac.h

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Can this be done via inheritance instead of copy? Electron got into a lot of trouble with copying old implementation and modifying.

Reply:
Our version is based on Sparkle framework via `SparkleGlue`.
Upstream is based on Keystone framework.
So, both would be completely different implementation.